src/chrome/content/rules/Blogtrottr.com.xml

```diff
@@ -0,0 +1,12 @@
+<!--
+	Connection closed:
+	- blog.blogtrottr.com
+-->
+
+<ruleset name="Blogtrottr.com">
+	<target host="blogtrottr.com" />
+	<target host="www.blogtrottr.com" />
+	<target host="static.blogtrottr.com" />
```

Comment:
Please also add `support.blogtrottr.com`

Reply:
Added, thank you.